glide.yaml

```diff
@@ -33,11 +32,11 @@ import:
 - package: github.com/apparentlymart/go-cidr
   version: v1.0.0
 - package: github.com/stretchr/testify
-  version: ~1.3.0
+  version: c679ae2cc0cb27ec3293fea7e254e47386f05d69
```

Comment:
glide complains that something require this SHA

Reply:
isn't there a specific version we can pin that will contain that commit?

Reply:
we don't use this in our code, looks like transitive dep that something else pin to that SHA, so I would not be concerned?

Reply:
That commit is in v1.2.2 and v1.3.0.